### PR TITLE
Write receipt to json file

### LINKED ISSUE #28: write `receipt.json`

### PR DESCRIPTION
**Pull request by kim201212**
close #28

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -125,9 +130,12 @@ program
 
         const prevContributions = await tryAndWait(sequencer, sessionID);
         const receipt = await runCeremony(sequencer, sessionID, entropy, prevContributions);
+        const receiptJson = JSON.stringify(receipt, null, '\t');
 
-        // TODO: Store this receipt?
-        console.log(receipt);
+        fs.writeFile('receipt.json', receiptJson, (err) => {
```
Comment (rootwarp) [inline index.js]: How do you think that using this routine as synchonously?
All process could be finished before the writing completed.

Comment (kim201212) [reply index.js]: Wirte file as synchonously 
https://github.com/dsrvlabs/czg-keremony/pull/29/commits/0259b8645162214fd4be6042a919a76527203627

**Comment by rootwarp** [comment]
@kim201212 

Last commit(fefc7210756962b069c1fd024510123cac786f4e) seems messy.
Would you mind squash last two commit?

**Comment by kim201212** [comment]
> @kim201212
> 
> Last commit([fefc721](https://github.com/dsrvlabs/czg-keremony/commit/fefc7210756962b069c1fd024510123cac786f4e)) seems messy. Would you mind squash last two commit?

squash commit! :)
https://github.com/dsrvlabs/czg-keremony/pull/29/commits/d6864776b57bbf16c4c45ab7205d79703242a485

File: index.js [hunk 2]
```diff
@@ -125,9 +130,12 @@ program
 
         const prevContributions = await tryAndWait(sequencer, sessionID);
         const receipt = await runCeremony(sequencer, sessionID, entropy, prevContributions);
+        const receiptJson = JSON.stringify(receipt, null, '\t');
 
-        // TODO: Store this receipt?
-        console.log(receipt);
+        fs.writeFilesy('receipt.json', receiptJson, (err) => {
```
Comment (rootwarp) [inline index.js]: Maybe this should be fixed like below.
```suggestion
        fs.writeFileSync('receipt.json', receiptJson, (err) => {
```

**Review by rootwarp — COMMENTED** [review]
@kim201212

File: index.js (hunk 2 above)
Comment (kim201212) [reply index.js]: [3cb4236](https://github.com/dsrvlabs/czg-keremony/pull/29/commits/3cb42361e44f4948b690a17511f17caf42f2cc67)

Resolved! :)

**Review by rootwarp — APPROVED** [review]
LGTM.
All good. it seems working well.

**Pull request merged**